src/main/java/net/nikr/eve/jeveasset/io/local/SettingsWriter.java

```diff
@@ -442,6 +446,39 @@ private void writeTablesViews(final Document xmldoc, final Map<String, Map<Strin
 		}
 	}
 
+	private void writeTablesFormula(final Document xmldoc, final Map<String, List<Formula>> formulas) {
+		Element tableFormulasNode = xmldoc.createElementNS(null, "tableformulas");
+		xmldoc.getDocumentElement().appendChild(tableFormulasNode);
+		for (Map.Entry<String, List<Formula>> entry : formulas.entrySet()) {
+			Element formulasNode = xmldoc.createElementNS(null, "formulas");
+			setAttribute(formulasNode, "tool", entry.getKey());
+			tableFormulasNode.appendChild(formulasNode);
+			for (Formula formula : entry.getValue()) {
+				Element formulaNode = xmldoc.createElementNS(null, "formula");
+				setAttribute(formulaNode, "name", formula.getColumnName());
+				setAttribute(formulaNode, "expression", formula.getOriginalExpression());
+				setAttributeOptional(formulaNode, "index", formula.getIndex());
+				formulasNode.appendChild(formulaNode);
+			}
+		}
+	}
+
+	private void writeTablesJumps(final Document xmldoc, final Map<String, List<Jump>> jumps) {
+		Element tableJumpsNode = xmldoc.createElementNS(null, "tablejumps");
+		xmldoc.getDocumentElement().appendChild(tableJumpsNode);
+		for (Map.Entry<String, List<Jump>> entry : jumps.entrySet()) {
+			Element jumpsNode = xmldoc.createElementNS(null, "jumps");
+			setAttribute(jumpsNode, "tool", entry.getKey());
+			tableJumpsNode.appendChild(jumpsNode);
+			for (Jump jump : entry.getValue()) {
+				Element jumpNode = xmldoc.createElementNS(null, "jump");
+				setAttribute(jumpNode, "systemid", jump.getSystemID());
+				setAttributeOptional(jumpNode, "index", jump.getIndex());
+				jumpsNode.appendChild(jumpNode);
+			}
+		}
+	}
+
```

Comment:
While I'd love to get it where all the settings for each table fall into a parent table element, instead of having a lot of lists with table indexes. I don't know that now is the time to start that.

Reply:
I feel like we're kind locked in on this, since we're going to load the old format anyway.
``net.nikr.eve.jeveasset.io.local.update` does allow you to modify the XML before loading it, so, I guess that is a possibility.
I feel like that is a bit out of scope for this feature, though. But, I do like the idea 👍

Reply:
Yeah, that's a big can of worms. Probably best to decide at some point to make a new version of the settings.xml and then create another version of the reader. When loading the file if version <= 2 (I think that's the version we're on now) then send it to the legacy reader if not use the new iteration.